workshop/content/exercises/install-tasks.adoc

```diff
@@ -25,48 +25,47 @@ spec:
+but tasks can have multiple steps. Since these steps run within the same pod,
 they have access to the same volumes in order to cache files, access configmaps,
-secrets, etc. `Tasks` can also receive inputs (e.g., a git repository) and produce
+secrets, etc. Tasks can also receive inputs (e.g. a git repository) and produce
 outputs (e.g., an image in a registry) in order to interact with each other.
 
-For each step as part of a `task`, an image is declared that will be used to host
+For each step as part of a task, an image is declared that will be used to host
 the step as it executes. In the example above, the image specified is `maven:3.6.0-jdk-8-slim`.
-The `maven:3.6.0-jdk-8-slim` image allows the step to execute a `mvn install` command
-to add an artifact to a local repository.
+The `maven:3.6.0-jdk-8-slim` image allows the step to run in a container a that provides
+necessary libraries to run a `mvn install` command to add an artifact to a local repository.
 
-As mentioned previously, only the requirement for a git repository is declared on
-the `task` above and not a specific git repository to be used. This allows `tasks`
-to be reusable for multiple `pipelines` and purposes. You can find more examples of
-reusable `tasks` in the link:https://github.com/tektoncd/catalog[Tekton Catalog]
+Only the requirement for a git repository is declared on the task under the `inputs:` property
+above and not a specific git repository to be used. This allows tasks to be reusable for multiple
+pipelines and purposes. You can find more examples of reusable tasks in the link:https://github.com/tektoncd/catalog[Tekton Catalog]
 and link:https://github.com/openshift/pipelines-catalog[OpenShift Catalog] repositories.
 
-Now that you understand the concept of a `task`, you should install the `s2i-nodejs`
+Now that you understand the concept of a task, you should install the `s2i-nodejs`
 and `openshift-client` tasks from the catalog repositories using `oc`. These will
-be the `tasks` you add to your `pipeline` for this tutorial.
+be the tasks you add to your pipeline for this tutorial.
 
-Create the `s2i-nodejs` `task` that will define and build a container image for a
+Create the `s2i-nodejs` task that will define and build a container image for a
 nodejs-ex application and push the resulting image to an image registry:
 
 [source,bash,role=execute-1]
 ----
 oc create -f tektontasks/s2i-nodejs-task.yaml
 ----
 
-Create the `openshift-client` `task` that will deploy the image created by `s2i-nodejs`
+Create the `openshift-client` task that will deploy the image created by `s2i-nodejs`
 as a container on OpenShift:
 
 [source,bash,role=execute-1]
 ----
 oc create -f tektontasks/openshift-client-task.yaml
 ----
 
-**NOTE**: For convenience, the `tasks` have been copied from their original locations
+**NOTE**: For convenience, the tasks have been copied from their original locations
```

Comment:
This is the admonition I was talking about earlier. Asciidoc supports a WARNING admonition, so try that for the error message note and see how it renders in the workshop itself. I'm hoping it's bright and red and obvious that users should read it :)